include/unistd.h

```diff
@@ -170,12 +170,14 @@ extern int tcsetpgrp(int, pid_t);
 extern char *getlogin_r(char *, int);
 #endif
 
-/* berkeley specific functions */
-extern int	getopt(int, char* const*, const char*);
-extern int	opterr;
-extern int	optind;
-extern int	optopt;
-extern char	*optarg;
+/* 1003.2-1992 */
+#ifndef _GETOPT_DECLARED
```

Comment:
Maybe better to name it something like HARVEY_GETOPT_DECLARED?

Reply:
POSIX compliance, that's for -D_GETOPT_DECLARED definition. Many configure scripts will check that.